src/style/design-tokens/carbon-scoped-tokens-provider/carbon-scoped-tokens-provider.component.js

```diff
@@ -21,32 +21,25 @@ const kebabCase = (input) =>
 export const tokensClassName = (theme) =>
```

Comment:
Is there any need to keep this anymore now that we're not adding the CSS variables to elements with this class?